src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,98 @@
+package tasks;
+
+import exceptions.DuckEmptyDeleteDescriptionException;
+import exceptions.DuckDeleteOutofBoundsException;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+public class Task{
+
+    private static final String LINE_SEPARATOR = "____________________________________________________________";
+    private static final String REMOVE_MESSAGE= "Noted. I've removed this task:";
+    protected static String description;
+    protected boolean isDone;
+    public static final Path FILE_PATH = Path.of("./data/duck.txt");
+
+    public Task(String description) {
+        Task.description = description;
+        this.isDone = false;
+    }
+
+    public static String getDescription() {
+        return description;
+    }
+
+    public void setDone(boolean done) {
+        isDone = done;
+    }
+
+    public String toString() {
+        String done = "[ ] ";
+        if (isDone) {
+            done = "[X] ";
+        }
+        return done + description;
+    }
+
+    public static void markTask(ArrayList<Task> tasks, String userInput, int index){
+        String[] split = userInput.split(" ");
+        int number = Integer.parseInt(split[1]);
```

Comment:
a suggestion: this variable 'number' could be named better (e.g. taskIndexToMark) for better readability